modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts

```diff
@@ -135,3 +135,39 @@ test('LPD-28908 update user information', async ({
 			)
 	).toBeVisible();
 });
+
+test('LPD-31020 Assign User', async ({
+	apiHelpers,
+	usersAndOrganizationsPage,
+}) => {
+	const user = await apiHelpers.headlessAdminUser.postUserAccount();
+	const organization = await apiHelpers.headlessAdminUser.postOrganization();
+
+	await apiHelpers.headlessAdminUser.assignUserToOrganizationByEmailAddress(
+		organization.id,
+		user.emailAddress
+	);
+
+	apiHelpers.data.push({
+		id: `${organization.id}_${user.emailAddress}`,
+		type: 'organizationUserAccountAssociation',
+	});
```

Comment:
I think we need to test this using UI and not API because probably the test is exactly to test the UI process